charts/civil-ccd/values.preview.template.yaml

```diff
@@ -32,26 +31,65 @@ civil-service:
 idam-pr:
   redirect_uris:
     XUI:
-      - https://xui-${SERVICE_NAME}.preview.platform.hmcts.net/oauth2/callback
+      - https://xui-${SERVICE_FQDN}/oauth2/callback
 
 civil-general-applications:
   enabled: true
 xui-webapp:
   nodejs:
-    ingressHost: xui-${SERVICE_NAME}.preview.platform.hmcts.net
+    ingressHost: xui-${SERVICE_FQDN}
 ccd:
   ccd-api-gateway-web:
     nodejs:
-      ingressHost: gateway-${SERVICE_NAME}.preview.platform.hmcts.net
+      ingressHost: gateway-${SERVICE_FQDN}
   ccd-data-store-api:
     java:
       environment:
         OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-{{ .Values.global.environment }}.internal:8443/openam/oauth2/hmcts
         CCD_S2S_AUTHORISED_SERVICES_CASE_USER_ROLES: civil_service,aac_manage_case_assignment
-      ingressHost: ccd-data-store-api-${SERVICE_NAME}.preview.platform.hmcts.net
+      ingressHost: ccd-data-store-api-${SERVICE_FQDN}
   ccd-definition-store-api:
     java:
-      ingressHost: ccd-definition-store-${SERVICE_NAME}.preview.platform.hmcts.net
+      ingressHost: ccd-definition-store-${SERVICE_FQDN}
+  am-role-assignment-service:
```

Comment:
is this pod required as in values.template.yaml  ..the data store alwyas point to aat role assignments

Reply:
Sorry didn't understand this is the definition store pod, which I believe is needed. Also as part of WA POC bit we haven't changed this.

Reply:
its line number   54 . . which introduced am role assignment service pod . .this was removed as we are using aat role assignments . . must have been introduced as part of wrong merge conflict fix from master

Reply:
Understood, thanks for pointing this out. I have removed it now. Build running